src/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import './index.css';
+import bookReducer from './reducers/book';
+import App from './components/app';
+
+const ids = [];
+
+const generateUniqueId = () => {
+  let number = Math.floor(Math.random() * 100);
+  while (ids.includes(number)) {
+    number = Math.floor(Math.random() * 100);
+  }
+  ids.push(number);
+  return number;
+};
+
+const initialsBooks = [{
+  id: generateUniqueId(),
+  title: 'Things fall apart',
+  category: 'Learning',
+},
+{
+  id: generateUniqueId(),
+  title: 'God Father',
+  category: 'History',
+}];
+
+const store = createStore(bookReducer, initialsBooks);
```

Comment:
What you pass to the store is not supposed to be the book reducer but a combined reducer where all the other reducers are put, that is why we were asked to make an index file in the reducers folder. if we have other reducers for other sections of the app, there is no way we can use them now, hence the use of combine is important for when there could be multiple reducers.

![Screenshot 2019-12-02 at 10 34 27 AM](https://user-images.githubusercontent.com/10922508/69948129-59874d00-14ef-11ea-8f05-11e299bdf75b.png)

Reply:
We've made this change